### ISSUE TITLE
Polymorph adding minion to graveyard

### ISSUE THREAD

**Issue by sdlorimor** [RedHatter/Graveyard#39]
Polymorph (hex, etc.) is bugging out with the graveyard mod.

For example, in this video Malygos had been polymorphed but was not actually killed.  The sheep was then killed, but the graveyard mod was putting Malygos in the graveyard list instead of sheep.  This also happens with shapechange spells such as Hex.

The sheep (frog, etc.) is only added to the graveyard list if the sheep is resurrected and killed.

**Comment by Paolkageist** [comment]
Unfortunately, this looks like a limitation of the deck tracker itself.  From the hdt_log.txt file (my opponent polymorphed my Dyn-o-matic then killed it with Arcane Missiles, and the deck tracker identified it as still being the Dyn-o-matic):  

`6:53:08 AM|Info|Player.Play >> [Opponent] id=57, cardId=CS2_022, cardName=Polymorph, zonePos=0,Info={turn=7, originalZone=DECK, hasOutstandingTagChanges=true, OriginalController=2}`

`6:53:11 AM|Info|Player.PlayToGraveyard >> [Opponent] id=57, cardId=CS2_022, cardName=Polymorph, zonePos=0,Info={turn=7, originalZone=DECK, OriginalController=2}`

`6:53:15 AM|Info|Player.Play >> [Opponent] id=54, cardId=EX1_277, cardName=Arcane Missiles, zonePos=0,Info={turn=7, originalZone=DECK, hasOutstandingTagChanges=true, OriginalController=2}`

`6:53:21 AM|Info|Player.PlayToGraveyard >> [Opponent] id=54, cardId=EX1_277, cardName=Arcane Missiles, zonePos=0,Info={turn=7, originalZone=DECK, OriginalController=2}`

`6:53:21 AM|Info|Player.PlayToGraveyard >> [Player] id=15, cardId=BOT_104, cardName=Dyn-o-matic, zonePos=0,Info={turn=7, mark=Created, created=true, originalZone=DECK, OriginalController=1}`

I'd have to look closer at what data HDT captures to see if it's fixable in this plugin or if it's something HDT itself needs to update.

**Comment by Drammg** [comment]
HDT don't actually recognize the shapecanged minion, i have an example with the card flavor text which don't work with evolved minions, i think it's related

**Comment by Supergecki** [comment]
Yeah, it's all tied to the same problem. Once a minion enters the battlefield, the Deck Tracker expects it to stay that minion until it leaves the battlefield.
